controllers/kogitoserverlessbuild_controller.go

```diff
@@ -85,31 +85,33 @@ func (r *KogitoServerlessBuildReconciler) Reconcile(ctx context.Context, req ctr
 	}
 
 	if phase == operatorapi.BuildPhaseNone {
-		if err := buildManager.Schedule(build); err != nil {
```

Comment:
on line 78, github didn't allow me to add the comment in that line.
Maybe we can remove the if from that line?

Reply:
Nice catch!